internal/graphapi/tools_test.go

```diff
@@ -110,12 +108,17 @@ func setupDB() {
 		errPanic("failed to start nats server", err)
 	}
 
-	natsClient, err := newNatsClient(ns)
+	pubcfg := events.PublisherConfig{
+		URL:    ns.ClientURL(),
+		Prefix: "com.infratographer",
```

Comment:
should the `Prefix` remain configurable via viper vs hard-coding here?

Reply:
I think it's fine to leave it hard-coded here as part of our tests.

Reply:
didn't see that test filename there. My bad